images/node/Dockerfile

```diff
@@ -17,6 +17,7 @@ ADD https://copr.fedoraproject.org/coprs/maxamillion/origin-next/repo/epel-7/max
 RUN yum install -y libmnl libnetfilter_conntrack openvswitch \
     libnfnetlink iptables iproute bridge-utils procps-ng ethtool socat openssl \
     binutils xz kmod-libs kmod sysvinit-tools device-mapper-libs dbus \
+    e2fsprogs xfsprogs ceph-common glusterfs-fuse iscsi-initiator-utils nfs-utils \
```

Comment:
We have this[1] bug that seems to indicate that's not sufficient, the last comment they've verified `glusterfs-fuse` is installed on the host. However I do know that e2fsprogs and xfsprogs were *not* installed in the particular image they mentioned so perhaps that's the real problem.

1 - https://bugzilla.redhat.com/show_bug.cgi?id=1289921